.stylelintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // See https://github.com/torchbox/stylelint-config-torchbox for rules.
+  extends: 'stylelint-config-torchbox',
```

Comment:
Should we use https://github.com/wagtail/stylelint-config-wagtail?

Reply:
Updated!